migrating_gazebo_classic_ros2_packages.md

```diff
@@ -0,0 +1,595 @@
+# Migrating ROS 2 packages that use Gazebo Classic
+
+The Gazebo simulator has its roots in the Gazebo Classic project, but it has a
+few significant differences that affect how a ROS 2 project uses the simulator.
+One difference is that ROS 2 projects now use the `ros_gz` package instead of
+`gazebo_ros_pkgs` as the source of launch files and other useful utilities.
+Another major difference is that `gazebo_ros_pkgs` provided a set of plugins
+that directly get loaded by Gazebo Classic and run as part of the simulation to
+provide an interface between ROS and Gazebo Classic. On the other hand, the
+primary mode `ros_gz` is used as bridge between ROS and gz-transport topics.
```

Comment:
```suggestion
Another major difference is that while [gazebo_ros_pkgs](https://github.com/ros-simulation/gazebo_ros_pkgs) provided a set of plugins
that directly get loaded by Gazebo Classic and run as part of the simulation to
provide an interface between ROS and Gazebo Classic,
[ros_gz](https://github.com/gazebosim/ros_gz) does not provide plugins but is used as bridge between ROS and gz-transport topics.
```

Reply:
In light of https://github.com/gazebosim/ros_gz/issues/490, I want to avoid saying `ros_gz` doesn't provide plugins.

Reply:
Reworded in 5547582b4d81e105ef572cd6969d0fc76ed2f941. What do you think?